tests/pubsub_logging_test.py

```diff
@@ -41,6 +46,105 @@ def test_compat_urlsafe_b64encode(self):
         self.assertEqual(expected, result)
 
 
+class GetPubsubClientTest(unittest.TestCase):
```

Comment:
I don't think these tests add any value, other than the least useful form of "coverage". Essentially, they boil down to "did our function wind up calling the other functions we listed in it", which I don't think is helping anybody.

I think test_get_pubsub_client_with_scopes should probably just use a typical service account key and assert that discovery.build gives us a useable pubsub client.

test_*_without_scopes may be a bit harder, because i'm not sure if you can easily generate a pre-scoped token outside the context of running on GCE.